app/config/request.js

```diff
@@ -116,16 +117,17 @@ export const deleteData = (uri, body)  => {
 }
 
 export const getCurrentLocation = () => {
+    Location.installWebGeolocationPolyfill()
```

Comment:
What is this doing for us?

Reply:
Geolocation doesn’t work in SDK 41 without that.

```
Remove expo-linear-gradient, expo-linking, expo-location, expo-permissions, and expo-sqlite dependencies from expo, along with related globals. As a side effect, navigator.geolocation is no longer automatically polyfilled unless the developer installs expo-location in the project. (#12097 by @brentvatne)```

https://github.com/expo/expo/blob/master/CHANGELOG.md

And the above is how I got it to work, but let me know if I didn’t do it in the best way!

Reply:
i'd move away from using `navigator.geolocation` if you use it, and instead just use `expo-location` methods through the apis they expose, eg:

```
import * as Location from 'expo-location';

// in your code
let location = await Location.getCurrentPositionAsync({});
```

Reply:
I appreciate your insights - thank you!

Also, I apologize for tagging you here! I had your changelog snippet as a codeblock, but I didn’t like how it was formatted so I edited and uncommented it and thus accidentally tagged you!!